packages/contracts/contracts/optimistic-ethereum/OVM/bridge/messaging/OVM_L1CrossDomainMessenger.sol

```diff
@@ -90,7 +142,8 @@ contract OVM_L1CrossDomainMessenger is iOVM_L1CrossDomainMessenger, Abs_BaseCros
         override
         public
         nonReentrant
-        onlyRelayer()
```

Comment:
I looked elsewhere: using brackets on modifiers with no args was inconsistent with other parts of the code.